src/strongbox/ui/cli.clj

```diff
@@ -720,6 +725,20 @@
     (half-refresh)))
 
 
+;;
+
+
+(defn-spec add-summary-to-user-catalogue nil?
```

Comment:
test

Reply:
can we rename it from 'add-summary-to-user-catalogue' to 'add-to-user-catalogue' ?